sympy/physics/continuum_mechanics/truss.py

```diff
@@ -198,46 +203,48 @@ def remove_node(self, label):
 
         >>> from sympy.physics.continuum_mechanics.truss import Truss
         >>> t = Truss()
-        >>> t.add_node('A', 0, 0)
+        >>> t.add_node(('A', 0, 0), ('B', 3, 0), ('C', 5, 0))
         >>> t.nodes
-        [('A', 0, 0)]
-        >>> t.add_node('B', 3, 0)
-        >>> t.nodes
-        [('A', 0, 0), ('B', 3, 0)]
-        >>> t.remove_node('A')
+        [('A', 0, 0), ('B', 3, 0), ('C', 5, 0)]
+        >>> t.remove_node('A', 'C')
         >>> t.nodes
         [('B', 3, 0)]
         """
-        for i in range(len(self.nodes)):
-            if self._node_labels[i] == label:
-                x = self._node_position_x[i]
-                y = self._node_position_y[i]
+        for label in args:
+            for i in range(len(self.nodes)):
+                if self._node_labels[i] == label:
+                    x = self._node_position_x[i]
+                    y = self._node_position_y[i]
 
-        if label not in self._node_labels:
-            raise ValueError("No such node exists in the truss")
+            if label not in self._node_labels:
```

Comment:
This condition has been used here too and in many more methods. The same suggestions can be applied here too.